api/config/cdr_versions_perf.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "cdrVersionId": 1,
+    "name": "Synthetic Dataset v1",
+    "dataAccessLevel": 1,
+    "bigqueryProject": "fc-aou-cdr-synthetic-test",
```

Comment:
I'd suggest pointing to all-of-us-ehr-dev for now, eventually we should probably create a separate CDR project for perf

Reply:
I can change that. Will do.